tests/DeploymentPipelineStepTest.php

```diff
@@ -102,6 +103,21 @@ public function testSnapshotSkipped() {
 	}
 
 	/**
+	 * Test snapshot is skipped upon user request.
+	 */
+	public function testSnapshotSkippedByUser() {
+		$step = $this->getDummyDeployment(false, true);
+		$step->start();
+
+		// Assert not error at startup
+		$this->assertEquals('Started', $step->Status);
+		$this->assertEquals('Deployment', $step->Doing);
+		$this->assertFalse(PipelineTest_MockLog::has_message('TestDeployStep:Snapshot creating snapshot of database'));
+		$snapshot = $step->Pipeline()->PreviousSnapshot();
+		$this->assertFalse($snapshot->exists(), 'No snapshot was created');
```

Comment:
Removed both occurences of confusing description. No other asserts in this test use descriptions either, so that's at least consistent.